openmetadata-ui/src/main/resources/ui/cypress/e2e/Features/updateDisplayName.spec.js

```diff
@@ -0,0 +1,278 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+// / <reference types="Cypress" />
+
+import {
+  interceptURL,
+  verifyResponseStatusCode,
+  visitEntityDetailsPage,
+} from '../../common/common';
+import {
+  createDataWithApi,
+  visitServiceDetailsPage,
+} from '../../common/serviceUtils';
+import { MYDATA_SUMMARY_OPTIONS } from '../../constants/constants';
+import {
+  DASHBOARD_DATA_MODEL,
+  DASHBOARD_SERVICE_API,
+  DATABASE_SERVICE_API,
+  DELETE_SERVICES,
+  ENTITIES_DISPLAY_NAME,
+  MESSAGING_SERVICE_API,
+  ML_MODAL_SERVICE_API,
+  PIPELINE_SERVICE_API,
+  SCHEMA_AND_DATABASE_DISPLAY_NAME,
+  SERVICES,
+  STORAGE_SERVICE_API,
+} from '../../constants/updateDisplayName.constant';
+
+const updateDisplayName = (displayName, apiPath) => {
+  if (apiPath) {
+    interceptURL('PATCH', apiPath, 'patchDisplayName');
+  }
+  cy.get('[data-testid="manage-button"]')
+    .should('exist')
+    .should('be.visible')
+    .click();
+
+  cy.get('[data-menu-id*="delete-button"]')
+    .should('exist')
+    .should('be.visible');
+  cy.get('[data-testid="rename-button"]').should('be.visible').click();
+
+  cy.get('#name').should('be.visible').should('be.disabled');
+  cy.get('#displayName').should('be.visible').should('not.be.disabled').clear();
+  cy.get('.ant-modal-footer').should('contain', 'Cancel');
+  cy.get('[data-testid="save-button"]').should('be.visible').click();
+  cy.get('.ant-modal-body').should('contain', 'Display Name is required');
+  cy.get('#displayName').type(displayName);
+  cy.get('[data-testid="save-button"]').should('be.visible').click();
+
+  if (apiPath) {
+    verifyResponseStatusCode('@patchDisplayName', 200);
+  }
+
+  cy.get('[data-testid="entity-header-display-name"]').should(
+    'contain',
+    displayName
+  );
+};
+
+describe('Update DisplayName', () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it('Preparing data', () => {
```

Comment:
It should be in `beforeAll` instead as a test

Reply:
I believe it should be part of the test, as we need login info + we are testing API calls, and if we move this to beforeAll, and something breaks there it will skip the all test.